test/uk/gov/hmrc/cardpaymentfrontend/services/ClientIdServiceSpec.scala

```diff
@@ -78,8 +79,9 @@ class ClientIdServiceSpec extends ItSpec {
         (TestJourneys.PfVat.journeyBeforeBeginWebPayment, ClientIds.VAEC),
         (TestJourneys.BtaVat.journeyBeforeBeginWebPayment, ClientIds.VAEC),
         (TestJourneys.VcVatReturn.journeyBeforeBeginWebPayment, ClientIds.VAEC),
-        (TestJourneys.VcVatOther.journeyBeforeBeginWebPayment, ClientIds.VAEC)
-
+        (TestJourneys.VcVatOther.journeyBeforeBeginWebPayment, ClientIds.VAEC),
+        (TestJourneys.Ppt.journeyBeforeBeginWebPayment, ClientIds.ETEC),
+        (TestJourneys.PfPpt.journeyBeforeBeginWebPayment, ClientIds.MIEC)
```

Comment:
is this correct? https://github.com/hmrc/pay-frontend/blob/9fdf06f45af5dd6e549d038b6e2674335a21940d/app/model/barclays/Merchants.scala#L94

Reply:
where is that so?